pythonforandroid/recipes/liblz4/__init__.py

```diff
@@ -0,0 +1,48 @@
+import os
+from os.path import join
+
+import sh
+
+from pythonforandroid.logger import shprint
+from pythonforandroid.recipe import Recipe
+from pythonforandroid.toolchain import current_directory
+
+
+class LibLZ4Recipe(Recipe):
+    name = 'liblz4'
+    version = '1.9.4'  # Use the desired version
+    url = 'https://github.com/lz4/lz4/archive/refs/tags/v{version}.tar.gz'
+    depends = []
+
+    def build_arch(self, arch):
+        env = self.get_recipe_env(arch)
+        build_dir = self.get_build_dir(arch.arch)
+        lib_dir = self.ctx.get_libs_dir(arch.arch)
+        include_dir = join(self.ctx.get_python_install_dir(), 'include', 'lz4')
```

Comment:
CI is failing with:
```
Traceback (most recent call last):
  File "/home/user/app/testapps/on_device_unit_tests/setup.py", line 106, in <module>
    setup(
  File "/home/user/app/venv/lib/python3.10/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.10/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.10/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.10/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/home/user/app/pythonforandroid/bdistapk.py", line 82, in run
    main()
  File "/home/user/app/pythonforandroid/entrypoints.py", line 18, in main
    ToolchainCL()
  File "/home/user/app/pythonforandroid/toolchain.py", line 677, in __init__
    getattr(self, command)(args)
  File "/home/user/app/pythonforandroid/toolchain.py", line 104, in wrapper_func
    build_dist_from_args(ctx, dist, args)
  File "/home/user/app/pythonforandroid/toolchain.py", line 163, in build_dist_from_args
    build_recipes(build_order, python_modules, ctx,
  File "/home/user/app/pythonforandroid/build.py", line 504, in build_recipes
    recipe.build_arch(arch)
  File "/home/user/app/pythonforandroid/recipes/liblz4/__init__.py", line 21, in build_arch
    include_dir = join(self.ctx.get_python_install_dir(), 'include', 'lz4')
TypeError: Context.get_python_install_dir() missing 1 required positional argument: 'arch'
```

Reply:
fixed

Reply:
Thanks, did you have a chance to test the recipe on-device, is it working?